typedoc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
I would move this file to a `config` directory to keep the configs in one place, separate from the scripts.